modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/BrokerAuthConfiguration.java

```diff
@@ -69,6 +80,42 @@ public void setEnabled(boolean enabled) {
         }
     }
 
+    /**
+     * Represents authorization configuration for broker
+     */
+    public static class AuthorizationConfiguration {
```

Comment:
We need the period after doc.  (Doc Dot please) :)

Reply:
Fixed in d1ed2c6